scene/resources/2d/tile_set.cpp

```diff
@@ -545,6 +545,16 @@ bool TileSet::has_source(int p_source_id) const {
 	return sources.has(p_source_id);
 }
 
+#ifdef TOOLS_ENABLED
+TileSetSource *TileSet::try_get_source(int p_source_id) const {
```

Comment:
I am not sure it is worth creating a new function just for that case, but well. I am not sure there's another way to do it.
I'd rather have another name, something like `get_source_no_error` or something like that, so it is prefixed the same way as `get_source`. Also, I would make it return a reference, like get_source does.

Reply:
Using references creates additional overhead. You can see it from this profile https://github.com/godotengine/godot/issues/85871#issuecomment-1846914890. 

Regarding the name of the method, I would change it to `get_source_ptr`.